Tools/LyTestTools/ly_test_tools/o3de/atom_tools_test.py

```diff
@@ -122,45 +122,45 @@ class MaterialEditorParallelTest(MaterialEditorSharedTest):
     is_parallelizable = True
 
 
-class MaterialEditorBatchedTest(MaterialEditorSharedTest):
+class AtomToolsBatchedTest(AtomToolsSharedTest):
     """
-    Test that will run serially batched with the tests in the same MaterialEditor instance,
-    though not executed in parallel with other MaterialEditor instances.
-    Reduces overhead from starting the MaterialEditor, while limiting side-effects between MaterialEditors.
+    Test that will run serially batched with the tests in the same Atom tools executable instance,
+    though not executed in parallel with other Atom tools executable instances.
+    Reduces overhead from starting the Atom tools executable, while limiting side-effects between executables.
 
     Does not support per test setup/teardown to avoid creating race conditions
     """
     is_batchable = True
     is_parallelizable = False
 
 
-class MaterialEditorTestSuite(MultiTestSuite):
+class AtomToolsTestSuite(MultiTestSuite):
     """
-    This class defines the values needed in order to execute a batched, parallel, or single MaterialEditor test.
+    This class defines the values needed in order to execute a batched, parallel, or single Atom tools executable test.
     Any new test cases written that inherit from this class can override these values for their newly created class.
     """
-    # Extra cmdline arguments to supply for every MaterialEditor for this test suite.
+    # Extra cmdline arguments to supply for every Atom tools executable for this test suite.
     global_extra_cmdline_args = ["-BatchMode", "-autotest_mode"]
     # Tests usually run with no renderer, however some tests require a renderer and will disable this.
     use_null_renderer = True
-    # Maximum time in seconds for a single MaterialEditor to stay open across the set of shared tests.
+    # Name of the Atom tools executable's log file.
+    log_name = "material_editor_test.log"  # We default to MaterialEditor for this suite.
```

Comment:
Remove the default values for these in the next update and error if the user doesn't explicit set it for the Atom tools executable tests.